src/server/main.py

```diff
@@ -19,9 +20,12 @@ def main():
     args = parser.parse_args()
 
     try:
+        now = datetime.now()
+        time_stamp = now.strftime("%m.%d.%Y_%H-%M-%S")
+        file_name = args.f + f'output_{time_stamp}.json'
         with open(args.file_path, 'r') as file:
             caption_data = json.load(file)
-            consume(caption_data)
+            consume(caption_data,file_name)
```

Comment:
You're passing in the file_name instead of the time_stamp.

Reply:
The file_name has the time_stamp appended to it